great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.navigation-bar {
+	display: flex;
+	padding: 10px;
+	justify-content: center;
+	/*width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;*/
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	background-color: white;
+	width: 100%;
+	align-items: center;
+	height: 36px;
+}
+
+header nav a {
+	vertical-align: middle;
+	margin: 10px 28px;
+	font-size: 15px;
+	text-decoration: none;
+	background-color: white;
+	color: lightgrey;
+}
+
+header nav a:first-of-type {
+	margin-left: 0;
+}
+
+header nav a:hover {
+	color: black;
+	font-size: 18px;
+	border-bottom: 2px dashed black;
+}
+
+
+.logo {
+	padding-left: 25px;
+	background-color: white;
+	/*justify-content: flex-end;*/
+	margin: 0 0 0 0;
+	vertical-align: middle;
+}
+
+h1 {
+	font-size: 70px;
+	text-align: center;
+}
+
+h3 {
+	font-size: 26px;
+	margin-bottom: 14px;
+}
+
+
+
+.first-section {
+	margin-top: 45px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.first-section button {
+	border: 1px solid black;
+	background-color: white;
+	padding: 5px 40px;
+	font-family: 'Titillium Web', sans-serif;
+	font-weight: 600;
+	font-size: 16px;
+	display: flex;
+	text-align: center;
+	align-items: center;
+	margin-left: 70px;
+}
+
+.left-side {
+	padding-top: 50px;
+}
+
+.second-section {
+	padding: 30px 0 50px 0;
+	margin-top: 50px;
+	border-top: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+}
+
+
+.middle-img {
+	width: 880px;
+}
+
+.third-group {
+	padding-top: 45px;
+	padding-bottom: 50px;
+	border-bottom: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+	padding-right: 15px;
+}
+
```

Comment:
I would like to see more use of comments to split your sections apart, this will make it easier to follow and know when something ends and something else begins.